src/ImageSharp/Common/Helpers/Numerics.cs

```diff
@@ -470,8 +470,8 @@ private static void ClampImpl<T>(Span<T> span, T min, T max)
         where T : unmanaged
     {
         ref T sRef = ref MemoryMarshal.GetReference(span);
-        var vmin = new Vector<T>(min);
-        var vmax = new Vector<T>(max);
+        Vector<T> vmin = new Vector<T>(min);
```

Comment:
Let's make sure we use target -typed-new expressions where possible.

Reply:
ok gonna look into configuring it and then change all places